WeightProducer/python/MCSamples_Summer20UL16.py

```diff
@@ -0,0 +1,105 @@
+import FWCore.ParameterSet.Config as cms
+
+from TreeMaker.WeightProducer.MCSample import MCSample
+
+# 13 TeV miniAOD samples - Summer20UL16
+Summer20UL16samples = [
+    # NB: amcatnlo samples have negative weight events, so NumberEvts = # positive - # negative
+    MCSample('WGJets_MonoPhoton_PtG-40to130_TuneCP5_13TeV-madgraph-pythia8', '106X_mcRun2_asymptotic_v17-v2', 'RunIISummer20UL16MiniAODv2', 'Constant', 1386165, False),
```

Comment:
one of the remaining todo items is to remove the `WrongPU` field from `MCSample` (that's the `False` at the end of this line)

Reply:
When I remove this, I'll also remove the 2017 METfix as it is no longer needed in UL per [1].

[1] https://twiki.cern.ch/twiki/bin/view/CMS/MissingETUncertaintyPrescription#Instructions_for_2017_data_with

Reply:
Yes, very good.

Reply:
Now addressed in ea9d776